### PR TITLE
Add wv2 flag `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs`

### PR DESCRIPTION
**Pull request by monica-ch**
Rendered article section for review:
* **WebView2 browser flags** > **Available WebView2 browser flags**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/webview-features-flags?branch=pr-en-us-3275#available-webview2-browser-flags - find "oopif"
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/webview2/concepts/webview-features-flags#available-webview2-browser-flags)
   * Added a new WebView2 feature flag `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` that we've shared with our 1P and 3P.

AB#54070651

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor
(after fixed Ref link & added other platform Ref links)

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ The following are some of the flags we've seen used.
 | `msEdgeDesignerDriverFix ` |  Use this flag to disable getting content and automatically showing the Designer feature. This feature is not supported in WebView2. |
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
 | `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging.  See [Remote debugging WebView2 WinUI 2 (UWP) apps](../how-to/remote-debugging.md), [Remotely debug Windows devices](../../devtools-guide-chromium/remote-debugging/windows.md). |
+| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, `AddWebResourceRequestedFilter` will also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:117]: does Win32 have a `AddWebResourceRequestedFilterWithRequestSourceKinds` method?  I only find a `AddWebResourceRequestedFilterWithRequestSourceKinds` method, by Searching Ref docs and by Find in "Overview of APIs".

present article section rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/webview-features-flags?branch=pr-en-us-3275#available-webview2-browser-flags - Find msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs

Overview of APIs, rendered:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3275&tabs=win32cpp#manage-network-requests-in-webview2 - find AddWebResourceRequestedFilter

@monica-ch @nishitha-burman

Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:117]: does this row apply to both of the overloads in .NET & WinRT?
I only linked to one of the two overloads of this method name.

Comment (monica-ch) [reply microsoft-edge/webview2/concepts/webview-features-flags.md:117]: Win32 has only AddWebResourceRequestedFilterWithRequestSourceKinds

**Review by monica-ch — COMMENTED** [review]
LGTM - Dev

File: microsoft-edge/webview2/concepts/webview-features-flags.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/webview-features-flags.md:117]: result:
```
If this flag is enabled, the `AddWebResourceRequestedFilter` method 
(including overloads) will also apply to out-of-process iframes.
```
@monica-ch @peiche-jessica 
Closing Comment.

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 2]
```diff
@@ -114,6 +114,7 @@ The following are some of the flags we've seen used.
...
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
 | `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging.  See [Remote debugging WebView2 WinUI 2 (UWP) apps](../how-to/remote-debugging.md), [Remotely debug Windows devices](../../devtools-guide-chromium/remote-debugging/windows.md). |
+| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, the `AddWebResourceRequestedFilter` method (including overloads) will also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:117]: alt design:
```suggestion
| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, the `AddWebResourceRequestedFilter` method (including overloads) will also apply to out-of-process iframes.  For API Reference links, see [Manage network requests in WebView2](./overview-features-apis.md#manage-network-requests-in-webview2) in _Overview of WebView2 features and APIs_. |
```
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3275#manage-network-requests-in-webview2
fyi @nishitha-burman, @monica-ch 

This alt design is:
* Compact - good for table cells, which are extremely restrictive for layout/Markdown.  solves problem of how to cram a tabset into almost mid-sentence compact form.
* Easy to tag & implement.  
* Lists related APIs; gives context, w/ lead-in in "Overview of APIs".

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/webview-features-flags.md:117]: continuing to consider this alt design, but closing Comment w/o now applying this alt design.

**Pull request merged**